arcade/experimental/examples/geometry_index_buffers.py

```diff
@@ -36,53 +35,82 @@ def __init__(self, width, height, title):
             void main() {
                 out_color = vec4(v_color, 1.0);
             }
-            """
+            """,
         )
 
         # Vertices
-        self.vertex_buffer = self.ctx.buffer(data=array('f',
-            [
-                # x  y    r    g    b
-                -1.0, -1.0, 1.0, 0.0, 0.0, # lower left
-                1, -1.0, 0.0, 1.0, 0.0, # lower right
-                1.0, 1.0, 0.0, 0.0, 1.0, # upper right
-                -1.0, 1.0, 0.0, 1.0, 1.0, # upper left
-                0, 0, 0, 0,0,  # dummy data for testing
-            ]
-        ))
-        self.ibo_8 = self.ctx.buffer(data=array('B', [3, 0, 2, 1]))
-        self.ibo_16 = self.ctx.buffer(data=array('H', [3, 0, 2, 1]))
-        self.ibo_32 = self.ctx.buffer(data=array('I', [3, 0, 2, 1]))
+        self.vertex_buffer = self.ctx.buffer(
+            data=array(
+                "f",
+                [
+                    # x  y    r    g    b
+                    -1.0,
+                    -1.0,
+                    1.0,
+                    0.0,
+                    0.0,  # lower left
+                    1,
+                    -1.0,
+                    0.0,
+                    1.0,
+                    0.0,  # lower right
+                    1.0,
+                    1.0,
+                    0.0,
+                    0.0,
+                    1.0,  # upper right
+                    -1.0,
+                    1.0,
+                    0.0,
+                    1.0,
+                    1.0,  # upper left
+                    0,
+                    0,
+                    0,
+                    0,
+                    0,  # dummy data for testing
+                ],
+            )
+        )
```

Comment:
Might need some cleanup here? I personally think a primitive per line is more readable in this case.